heron/scheduler-core/src/java/com/twitter/heron/scheduler/dryrun/FormatterUtils.java

```diff
@@ -63,6 +69,14 @@ private FormatterUtils() {
     STRIKETHROUGH
   }
 
+  public enum ContainerChange {
+    UNAFFECTED,
+    MODIFIED,
```

Comment:
it is from [here](https://github.com/twitter/heron/pull/1788/files/25a39b507aa8c58419658b335289711afdf7c513#diff-4850d7a660ab463553d7e473b7ee6142L42). Modified means there are instances added/removed from the container.